internal/command/e2etest/automation_test.go

```diff
@@ -243,3 +243,42 @@ func TestPlanOnlyInAutomation(t *testing.T) {
 		t.Error("plan file was created, but was not expected")
 	}
 }
+
+func TestPlanOnDeprecated(t *testing.T) {
+	t.Parallel()
+
+	// This test reaches out to registry.opentofu.org to download the
+	// template and null providers, so it can only run if network access is
+	// allowed.
+	skipIfCannotAccessNetwork(t)
```

Comment:
It doesn't seem like the `deprecated-values` test fixture actually uses either of the providers mentioned in the comment above, or anything else that would be loaded over the network, so I think we could remove this guard to allow this test to run unconditionally.

Reply:
Yep, good catch. Copy-paste leftover.
Fixed in 538dec6f1ad081f9b78bcbb614cf595869b21a6f